### DISCUSSION TITLE
无法正确弹出主题

### DISCUSSION THREAD

**Discussion by energy718** [vuepress-theme-hope/vuepress-theme-hope#1943] [Q&A] [answered]
使用create vuepress-theme-hope 创建模板项目 执行模板生成的主题弹出指令失败
![image](https://user-images.githubusercontent.com/45483349/171617202-4e22d490-965c-440a-a50b-e7f8ea76da12.png)

**Comment by Mister-Hope** [comment, marked as answer]
VuePress 1 is no longer maintained, and this is not v1 repo.

See https://github.com/vuepress-theme-hope/vuepress-theme-hope/discussions/1535

And if you have futher questions, take actions in https://github.com/vuepress-theme-hope/vuepress-theme-hope-v1/
